html/Ellipsis/style.css

```diff
@@ -0,0 +1,490 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Lato", sans-serif;
+  font-weight: lighter;
+}
+
+html {
+  font-size: 0.9rem;
+}
+
+@media screen and (min-width: 768px) {
+  html {
+    font-size: 1rem;
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  html {
+    font-size: 1.1rem;
+  }
+}
+
+@media screen and (min-width: 1600px) {
+  html {
+    font-size: 1.2rem;
+  }
+}
+
+/* ------------------------------------------------ Banner ------------------------------------------------ */
+.banner {
+  padding: 150px 0 50px 0;
+  background: linear-gradient(rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.2)),
+    url("./assets/banner/layer-8-min.png") no-repeat right center;
+  background-size: cover;
+  color: #ffffff;
+  text-align: center;
+}
+
+.banner__header {
+  width: 60%;
+  margin: 0 auto;
+}
+
+.banner__title {
+  margin: 40px 0 20px 0;
+  font-size: 1.7rem;
+  font-weight: bold;
+  text-transform: uppercase;
+}
+
+.banner__description {
+  margin-bottom: 100px;
+}
+
+.banner__btn {
+  background: transparent;
+  border: none;
+  cursor: pointer;
+}
+
+.banner__mouse {
+  width: 20px;
+}
+
+@media screen and (min-width: 768px) {
+  .banner {
+    background: linear-gradient(rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.2)),
+    url("./assets/banner/layer-8@2x-min.png") no-repeat right center;
+    background-position: right top;
+    background-size: cover;
+  }
+
+  .banner__shape > img {
+    width: 50px;
+  }
+
+  .banner__title {
+    font-size: 2rem;
+  }
+
+  .banner__description{
+    width: 70%;
+    margin: 40px auto 100px auto; 
+  }
+
+  .banner__mouse {
+    width: 25px;
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  .banner{
+    background: linear-gradient(rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.2)),
+    url("./assets/banner/layer-8@3x-min.png") no-repeat right center;
+    background-position: right top;
+    background-size: 100% 100%;
+  }
+
+  .banner__shape {
+    margin-left: -20px;
+  }
+
+  .banner__title {
+    font-size: 2.5rem;
+  }
+
+  .banner__description{
+    margin: 40px auto 250px auto; 
+  }
+}
+
+
+/* ------------------------------------------------ About us ------------------------------------------------ */
+.about-us {
+  position: relative;
+  padding: 70px 10%;
+  text-align: center;
+}
+
+.about-us__title {
+  position: absolute;
```

Comment:
For every section (about us, services, what to do etc) you reinvent title styling — not scalable solution. Please think how to make it universal